src/components/H/H3.js

```diff
@@ -6,9 +6,10 @@ import get from 'extensions/themeGet';
 const H3 = styled.h3.withConfig({ displayName: 'H3' })`
   color: ${get('colors.text.default')};
   font-weight: 300;
-  font-family: ${get('fonts.brand')};
+  font-family: 'Overpass', sans-serif;
```

Comment:
Let's keep this controlled by the brand font in the theme, in case we need to update the font across the library later.

Reply:
brand fonts have more than what is needed here. Does it matter?

Reply:
Any font after the first one is just a fallback, so it won't make any difference if things go well.